x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/content.tsx

```diff
@@ -53,6 +58,10 @@ export const AttacksPageContent = React.memo(({ dataView }: AttacksPageContentPr
     setShowFlyout(true);
   }, []);
   const onClose = useCallback(() => setShowFlyout(false), []);
+  const [assignees, setAssignees] = useState<AssigneesIdsSelection[]>([]);
```

Comment:
Like mentioned in another comment, I think we should remove all the code related to assignees for now